ast/src/main/java/com/graphicsfuzz/common/ast/TranslationUnit.java

```diff
@@ -148,6 +148,16 @@ public void addDeclarationBefore(Declaration newDecl, Declaration existingDecl)
     throw new IllegalArgumentException("Existing declaration not found.");
   }
 
+  public void updateTopLevelDeclaration(Declaration newDecl, Declaration existingDecl) {
```

Comment:
As you have added some public methods to TranslationUnit, please add corresponding tests to TranslationUnitTest.

Reply:
Done.